File: JavaScript Solutions/miniMaxSum.js
'use strict';

process.stdin.resume();
process.stdin.setEncoding('utf-8');

let inputString = '';
let currentLine = 0;

process.stdin.on('data', function(inputStdin) {
    inputString += inputStdin;
});

process.stdin.on('end', function() {
    inputString = inputString.split('\n');

    main();
});

function readLine() {
    return inputString[currentLine++];
}

/*
 * Complete the 'miniMaxSum' function below.
 *
 * The function accepts INTEGER_ARRAY arr as parameter.
 */

function miniMaxSum(arr) {
    let totalSum = arr.reduce((total, num) => total + num, 0)
    let resultOfSums = []
    
    arr.forEach( val => resultOfSums.push(totalSum - val) )
    resultOfSums.sort((a, b) => a - b )
    
    console.log(`${resultOfSums[0]} ${resultOfSums[4]}`)
}

File: Python Solutions/miniMaxSum.py
#!/bin/python3

import math
import os
import random
import re
import sys

#
# Complete the 'miniMaxSum' function below.
#
# The function accepts INTEGER_ARRAY arr as parameter.
#

def miniMaxSum(arr):
    result = sum(arr)
    resultOfSums = []
    
    for num in arr:
        resultOfSums.append(result - num)
    resultOfSums.sort()
    print(f'{resultOfSums[0]} {resultOfSums[4]}')
    

if __name__ == '__main__':

    arr = list(map(int, input().rstrip().split()))

    miniMaxSum(arr)